bots/notify/src/test/java/org/openjdk/skara/bots/notify/comment/CommitCommentNotifierTests.java

```diff
@@ -165,4 +165,61 @@ void testCommitCommentWithIssues(TestInfo testInfo) throws IOException {
             assertEquals("- [" + issue.id() + "](" + issue.webUrl() + ")", lines[7]);
         }
     }
+
+    /**
+     * Test that the CommitCommentNotifier never repeates the exact same comment
+     */
+    @Test
+    void testNoRepeatedCommitComment(TestInfo testInfo) throws IOException {
+        try (var credentials = new HostCredentials(testInfo);
+             var tempFolder = new TemporaryDirectory()) {
```

Comment:
Minor: indentation is different from below lines.

Reply:
Yes, that seems to be a recurring style choice in the codebase. The contents of the try () is lined up with the start of the opening parentheses.